### DISCUSSION TITLE
小红书图文创作——GPTs队

### DISCUSSION THREAD

**Discussion by Star-cre** [InternLM/Tutorial#278] [project]
### 构建小红书图文创作应用的目的：

> **提高效率**：通过大模型的语言生成能力，帮助用户快速生成高质量的图文内容，节省用户的时间和精力，同时提高内容创作的效率。
**个性化推荐**：利用大模型对用户喜好和行为的分析，可以根据用户的偏好生成个性化推荐的内容，提升用户体验，增加用户粘性。
**降低门槛**：对于不擅长创作或没有创作经验的用户，自动化图文生成应用可以降低创作门槛，让更多人参与到内容创作和分享中来。
**增加多样性**：通过大模型的丰富知识库和创作能力，可以生成多样性的内容，丰富平台上的图文信息，满足用户多样化的需求。

### 计划设计模块：

- 个人赛道定位模块：通过开放式问题引导用户，帮助用户找到喜欢的方向并确定选题。（计划采用Prompt模板来引导模型完成）
- 交互生成图文模块：根据用户喜欢的选题给出参考意见，并于用户交互进行确认。（计划采用stableDiffusion/mj+InternLM7B，为了达成图文一致，或采取其他模型进行意图识别分析，搜集小红书文章，使用xutuner进行模型的微调。）

### 项目成员：

- [Star-cre](https://github.com/Star-cre)
- [Aitejiu](https://github.com/Aitejiu)
- [2404589803](https://github.com/2404589803)
- [Wly0910](https://github.com/Wly0910)
- [Durian-1111](https://github.com/Durian-1111)

**Comment by Aitejiu** [comment]
你好你好！

**Comment by 2404589803** [comment]
各位大佬好

**Comment by Wly0910** [comment]
Great Job!

**Comment by Star-cre** [comment]
欢迎大家提出优化意见！

**Comment by Durian-1111** [comment]
来啦

**Comment by SiliconSisyphus** [comment]
请问队伍里人满了

**Reply by 2404589803** [reply]
满人了，实在抱歉😢
